client/blocks/login/two-factor-authentication/two-factor-actions.jsx

```diff
@@ -44,6 +44,15 @@ class TwoFactorActions extends Component {
 
 		this.props.switchTwoFactorAuthType( 'authenticator' );
 	};
+
+	recordBackupLinkClick = ( event ) => {
+		event.preventDefault();
```

Comment:
I see this handler is attached to the `onClick` event of a button without a `href`. Is `event.preventDefault()` necessary then?